include/deal.II/fe/fe_values.h

```diff
@@ -3757,6 +3757,16 @@ class FEFaceValues : public FEFaceValuesBase<dim, spacedim>
                                             level_dof_access>> &cell,
          const unsigned int                                     face_no);
 
+  /**
+   * Alternative reinitialization function that takes an iterator to the
+   * current face instead of its number.
+   */
+  template <template <int, int> class DoFHandlerType, bool level_dof_access>
+  void
+  reinit(const TriaIterator<DoFCellAccessor<DoFHandlerType<dim, spacedim>,
+                                            level_dof_access>> &   cell,
+         const TriaIterator<TriaAccessor<dim - 1, dim, spacedim>> &face);
+
   /**
    * Reinitialize the gradients, Jacobi determinants, etc for the given face
```

Comment:
While you're here anyway, would you indulge me in making the following changes:
```suggestion
   * Reinitialize the gradients, Jacobi determinants, etc., for the given face
```